kn/fotos/api.py

```diff
@@ -42,7 +42,8 @@ def album_json(album, user):
         current_album = current_album.get_parent()
 
     return {'children': json_children,
-            'parents': json_parents}
+            'parents': json_parents,
+            'visibility': album.visibility[0]}
```

Comment:
Hier lijk je ervan uit te gaan dat visibility altijd één element is.  Misschien is het beter dat dan overal door te voeren.  Dat is duidelijker.